checker/src/main/java/org/checkerframework/checker/determinism/DeterminismAnnotatedTypeFactory.java

```diff
@@ -883,6 +883,78 @@ public AnnotationMirror getNewClassAnnotation(NewClassTree tree) {
     }
 
     class DeterminismQualifierHierarchy extends GraphQualifierHierarchy {
+        @Override
+        public AnnotationMirror leastUpperBound(AnnotationMirror a1, AnnotationMirror a2) {
```

Comment:
@smillst I don't see any difference between the subtyping relationship and LUB for polymorphic qualifiers. Since the leastUpperBound() method in MultiGraphHierarchy calls isSubtype(), do we still need to override leastUpperBound() here?

Reply:
`@PolyDet("down")` and `@PolyDet("noOrderNonDet")` are unrelated, but their lub is `@PolyDet`.

Reply:
Got it. Thanks.